scripts/triggers/systems/dnd5e.mjs

```diff
@@ -0,0 +1,191 @@
+import { TRIGGERS } from '../../triggers.mjs'
+
+export default function init() {
+  DND5ETriggers.init();
+}
+
+export class DND5ETriggers {
+  /**
+   * Initialize the submodule.
+   */
+  static init() {
+    if (game.system.id !== "dnd5e") return;
+
+    // Inject our triggers into existing categories
+    const effectTriggers = TRIGGERS.find(t => t.label === "EFFECTMACRO.EffectTriggers");
+    const combatantTriggers = TRIGGERS.find(t => t.label === "EFFECTMACRO.CombatantTriggers");
+
+    effectTriggers.options.push(
+      "dnd5e.applyDamage",
+      "dnd5e.shortRest",
+      "dnd5e.longRest",
+      "dnd5e.beginConcentrating",
+      "dnd5e.endConcentration",
+      "dnd5e.rollHitDie"
+    );
+
+    combatantTriggers.options.push(
+      "dnd5e.rollAttack",
+      "dnd5e.rollDamage",
+      "dnd5e.rollInitiative"
+    );
```

Comment:
Same here, they might seem related to what a combatant can do in combat, but these are more dnd5e centric than the generic Combatant triggers already in the module.